announcer.go

```diff
@@ -224,11 +230,12 @@ func makeRDSMessage(currentTrack track) string {
 	if len(baseMessage) + len(stationID) < 128 {
 		totalMessage = baseMessage + stationID
 	}
+	writemsg(fmt.Sprintf("Sending to RDS: %+s", "DPS="+totalMessage + "\n"), verbose);
```

Comment:
Nit: I don't think that `makeRDSMessage()` should be logging a message about _sending_ that message.